xwayland/xwm.c

```diff
@@ -244,6 +250,24 @@ static void xwm_set_net_client_list(struct wlr_xwm *xwm) {
 			XCB_ATOM_WINDOW, 32, mapped_surfaces, windows);
 }
 
+static void xwm_set_net_client_list_stacking(struct wlr_xwm *xwm) {
+	size_t num_surfaces = wl_list_length(&xwm->surfaces_in_stack_order);
+	xcb_window_t windows[num_surfaces + 1];
+
+	// We store surfaces in top-to-bottom order because this is easier to reason
```

Comment:
Ah. This is inconsistent with the rest of wlroots: subsurfaces and popups are ordered in the other order, ie. the first item is underneath other items.

Maybe we should consider doing that too for Xwayland for consistency?

Reply:
I removed the comment, because it's no longer relevant.